core/Crossovers/BlendCrossoverAlfa.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+from core import Crossover, Population
```

Comment:
Shouldn't it be `from core.Crossover import Crossover` and the same goes for Population?